server/webapp/WEB-INF/rails.new/app/views/new_navigation_elements/_footer.html.erb

```diff
@@ -22,4 +22,9 @@
             </span>
       </div>
   </div>
+  <script type="application/javascript">
+    require(['models/shared/version_updater'], function(VersionUpdater) {
```

Comment:
Additionally the `check_go_updates?` guard exists on old pages, but not in this one?